resources/config/services/services.yml

```diff
@@ -21,7 +21,7 @@ services:
       $applicationEmail: '%application.email%'
       $applicationEndDate: '%application.enddate%'
       $applicationTitle: '%application.title%'
-      $resetForm: '@OpenCFP\Http\Form\ForgotForm'
```

Comment:
The problem here is that we want to add a binding for the *actual* `$resetForm` in #1009, and as the name of the form type suggests, this is actually a forgot form (even if that name is a bit awkward).